bokehjs/src/coffee/models/plots/plot.coffee

```diff
@@ -163,8 +164,7 @@ class PlotView extends Renderer.View
     if @mget('responsive')
       throttled_resize = _.throttle(@resize, 100)
       $(window).on("resize", throttled_resize)
-      # Just need to wait a small delay so container has a width
-      _.delay(@resize, 10)
+      @resize()
 
```

Comment:
Good catch. I was going to complain that Sarah probably added this for a reason, but we cover this with our checking-whether-its-in-the-dom-and-recalling-if-not thing.